projects/mysadna/src/app/components/user-details/user-details.component.html

```diff
@@ -1,51 +1,47 @@
-<div *ngIf="user" class="user-details">
-  <h1 class="user-header">{{user.getFirstName()}} {{user.getLastName()}}</h1>
-  <div class="user-container">
-    <div class="left-container">
-      <div class="user-image-container">
-        <img src="{{user.getUserImg()}}" class="user-image">
-      </div>
-      <p class="user-name">
-        {{user.getUserName()}}
-      </p>
-      <p class="email">
-        {{user.getEmail()}}
-      </p>
-      <ul class="social-networks">
-        <li *ngFor="let socialNetwork of user.getSocialNetworkList()" class="social-network">
-          <div class="{{socialNetwork}}">
-            <img src="./assets/images/{{socialNetwork}}.svg" class="{{socialNetwork}}">
-          </div>
-        </li>
-      </ul>
-    </div>
-    <div class="right-container">
-      <div class="projects">
-        <h3 class="projects-header">Projects:</h3>
-        <ul class="projects-list">
-          <li *ngFor="let project of userProjects">
-            <a [routerLink]="'/project-details/' + project.getProjectId()" class="project">
-              {{project.getProjectName()}}
+<div class="user-details" *ngIf="user">
+  <div class="media">
+    <div class="user-header">{{ user.getFirstName() }} {{ user.getLastName() }}</div>
+    <div class="user-container media-row-col">
+      <div class="user-label">
+        <div class="flex">
+          <img [src]="user.getUserImg()">
```

Comment:
Instead of using expression binding (`{{ expression }}`), we can bind whole attribute: `[src]`.

Reply:
Noted.